gallery/src/demos/demo-hui-glance-card.js

```diff
@@ -118,7 +118,8 @@ const CONFIGS = [
     - entity: lock.kitchen_door
       tap_action: toggle
     - entity: light.ceiling_lights
-      tap_action: turn-on
+      tap_action: call-service
+      service: light.turn_on
```

Comment:
Should we make this light.toggle?

Reply:
If you prefer it, bit for toggle we already have tap_action: toggle